_layouts/home.html

```diff
@@ -1,57 +1,50 @@
 <!doctype html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link href="https://fonts.googleapis.com/css?family=Vollkorn" rel="stylesheet">
-    <link rel="stylesheet" href="/css/style.css">
-    <title>{{site.title}}</title>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Vollkorn" rel="stylesheet">
+  <link rel="stylesheet" href="/css/style.css">
+  <link rel='shortcut icon' type='image/x-icon' href='/favicon.ico'/>
+  <title>{{site.title}}</title>
 </head>
 <body>
-<header style="background: #fff;" class="pdb-50">
-    {% include top-bar.html %}
-    <div class="container">
-        <div class="columns">
-            <div class="column pdt-50">
-                <span class="icon has-text-info">
-                  <i class="fab fa-twitter"></i>
-                </span>
-                <span class="icon has-text-grey-dark">
-                  <i class="fab fa-github"></i>
-                </span>
-                <span class="icon has-text-warning">
-                  <i class="fas fa-rss"></i>
-                </span>
+<header class="pdb-50">
+  {% include top-bar.html %}
+  <div class="container">
+    <div class="columns">
+      <div class="column pdt-50">
+        <div class="navbar-start">
+          <div class="navbar-item">
+            <div class="field is-grouped">
+              {% include social-links.html %}
```

Comment:
let's keep positioning of social-links to the right to make it consistent.
maybe add waving gif flag of nepal instead?

Reply:
Yeah! Great idea. I will do that today :)

Reply:
@zoras  dai, After Unboxing, blog page looks like this !

<img width="1260" alt="screen shot 2018-04-19 at 8 56 01 pm" src="https://user-images.githubusercontent.com/8276878/39000598-2112004e-4414-11e8-8c46-8127724ee00a.png">


I think in the post page, in right side related articles, archives, categories widget are required. what do you suggest ? post share feature is also remaining feature.

Reply:
Nice.
1. Jekyll has inbuilt method called [site.related_posts](https://jekyllrb.com/docs/variables/#site-variables) to show related post, 
2. Or we can use tags https://github.com/toshimaru/jekyll-tagging-related_posts
3. https://rgardler.github.io/2015/07/28/adding-related-posts-to-jekyll-blog
4. For post sharing feature use this https://github.com/mmistakes/so-simple-theme/blob/master/_includes/social-share.html